tests/airlock.test.js

```diff
@@ -1,16 +1,81 @@
 const axios = require("axios");
+const utils = require("./utils/utils");
 
-const { SERVER_URL = "http://localhost:8000/api" } = process.env;
+const {
+    SERVER_URL = "http://localhost:8000/api",
+    DEFAULT_INVALID_TEST_TOKEN = "INVALID_TOKEN"
+} = process.env;
+
+const axiosConfig = {
```

Comment:
all the tests will be sharing the same axiosConfig, even though some test cases use a different value for the Authorization field. So if test's were running concurrently or order wasn't guaranteed then once test would find the test data in a different state than expected.

Let's move this `const axiosConfig = {..}` inside the relevant `describe()`:
- line 38 before `it("Then replies with a status code 200"`
- line 47 before `it("Then replies with a status code 403"`
- line 69 before the `beforeAll`